client.bal

```diff
@@ -0,0 +1,478 @@
+        }
+        return getFileById(self.httpClient , fileId, optional);
+    }
+
+    # Download file using the fileID.
+    # 
+    # + fileId - ID of the file to retreive
+    # + return - If successful, returns `string`. Else returns `error`
+    remote function downloadFile(string fileId) returns @tainted string|error {
+        GetFileOptional optional = {supportsAllDrives : true, fields : WEB_CONTENT_LINK};
+        File fileResponse = check getFileById(self.httpClient , fileId, optional);
+        return fileResponse?.webContentLink.toString();
+    }
+
+    # Retrieve files.
+    # 
+    # + optional - 'ListFilesOptional' used to add query parameters to the request
+    # + return - If successful, returns stream of files `stream<File>`. Else returns `error`
+    remote function getFiles(ListFilesOptional? optional = ()) returns @tainted stream<File>|error {
+        return getFiles(self.httpClient, optional);
+    }
+
+    # Filter and retreive files using filter string
+    # 
+    # + filterString - Query used to find what you need. Read documentation for query string patterns.
+    # + noOfFiles - Number of files to retreive 
+    # + orderBy - A comma-separated list of sort keys. Valid keys are 'createdTime', 'folder', 'modifiedByMeTime', 
+    #             'modifiedTime', 'name', 'name_natural', 'quotaBytesUsed', 'recency', 'sharedWithMeTime', 'starred', 
+    #              and 'viewedByMeTime'
+    # + return - If successful, returns stream of files `stream<File>`. Else returns `error`
+    remote function filterFiles(string filterString, int? noOfFiles = (), string? orderBy = ()) returns @tainted 
+                                stream<File>|error {
+        ListFilesOptional optional = {};
+        optional.q = filterString;
+        optional.supportsAllDrives = true;
+        optional.includeItemsFromAllDrives = true;
+        if (noOfFiles is int){
+            optional.pageSize = noOfFiles;
+        }
+        if (orderBy is string){
+            optional.orderBy = orderBy;
+        }
+        return getFiles(self.httpClient, optional);
+    }
+
+    # Retrieve files by Name
+    # 
+    # + fileName - Name of the file to search (Partial search)
+    # + noOfFiles - Number of files to retreive 
+    # + orderBy - A comma-separated list of sort keys. Valid keys are 'createdTime', 'folder', 'modifiedByMeTime', 
+    #             'modifiedTime', 'name', 'name_natural', 'quotaBytesUsed', 'recency', 'sharedWithMeTime', 'starred', 
+    #              and 'viewedByMeTime'
+    # + return - If successful, returns stream of files `stream<File>`. Else returns `error`
+    remote function getFilesByName(string fileName, int? noOfFiles = (), string? orderBy = ()) 
+                                    returns @tainted stream<File>|error {
+        ListFilesOptional optional = {};
+        string searchString = NAME + SPACE + CONTAINS + SPACE + SINGLE_QUOTE + fileName + SINGLE_QUOTE + SPACE + AND + 
```

Comment:
Always split the line before an operator
Refer: https://ballerina.io/learn/user-guide/coding-conventions/#line-breaks
check all places

Reply:
Thanks Sachini.
Made the changes